converter/src/main/java/gov/cms/qpp/conversion/util/MeasureConfigHelper.java

```diff
@@ -0,0 +1,25 @@
+package gov.cms.qpp.conversion.util;
+
+import gov.cms.qpp.conversion.model.Node;
+import gov.cms.qpp.conversion.model.validation.MeasureConfig;
+import gov.cms.qpp.conversion.model.validation.MeasureConfigs;
+
+public class MeasureConfigHelper {
+
+	public static final String MEASURE_ID = "measureId";
+
+	private MeasureConfigHelper() {
+		// private for this helper class
+	}
+
+	/**
+	 * Convenience method to retrieve the measure configuration for validation from an ecqm node
+	 *
+	 * @param node Contains the id that associates with the measure config
+	 * @return
+	 */
+	public static MeasureConfig getMeasureConfig(Node node) {
```

Comment:
What was your thoughts on rolling this in to `MeasureConfigs` (notice the `s`) class?
Granted, I don't think I would want `MeasureConfigs` to know about `Node`s...

Reply:
Initially I thought about that, however, I liked the idea of passing in a node to reduce some code duplication. I also didn't want `MeasureConfigs` to know about the `Node` class.